Core.PoseEditor/AMModules/BonesEditor.cs

```diff
@@ -288,6 +289,10 @@ public override void GUILogic()
             }
             GUILayout.EndHorizontal();
             _boneEditionScroll = GUILayout.BeginScrollView(_boneEditionScroll, GUI.skin.box, GUILayout.ExpandHeight(true));
+
+            if (_search != oldSearch || _search == null)
```

Comment:
_search can never be null (if it was the UI would break), so it only needs a Length == 0 check.